pacman-nodejs-app/docs/pacman-nodejs-app-aws-migration.md

```diff
@@ -0,0 +1,532 @@
+
+```bash
+kubectl create -f deployments/pacman-deployment-rs.yaml
+```
+
+Scale the deployment, since the deployment definition has replicas set to 0.
+
+```bash
+kubectl scale deploy/pacman --replicas=2
+```
+
+Verify the containers have been created and are in the running state:
+
+```bash
+kubectl get pods -o wide --watch
+```
+
+#### Save Pac-Man Load Balancer IP
+
+```bash
+PACMAN_SRC_PUBLIC_IP=$(kubectl get svc pacman --output jsonpath="{.status.loadBalancer.ingress[0].hostname}".)
+
+```
+
+#### Add DNS A record
+
+Set the value of your `ZONE_NAME` and `DNS_NAME` name used for your Google Cloud DNS configuration.
+
+```bash
+ZONE_NAME=zonename
+DNS_NAME=example.com.
+```
+
+Then execute the below commands:
+
+```bash
+gcloud dns record-sets transaction start -z=${ZONE_NAME}
+gcloud dns record-sets transaction add -z=${ZONE_NAME} --name="pacman.${DNS_NAME}" --type=CNAME --ttl=1 "${PACMAN_SRC_PUBLIC_IP}"
+gcloud dns record-sets transaction execute -z=${ZONE_NAME}
+```
+
+## Play Pac-Man
+
+Go ahead and play a few rounds of Pac-Man and invite your friends and colleagues by giving them your FQDN to your Pac-Man application
+e.g. [http://pacman.example.com/](http://pacman.example.com/) (replace `example.com` with your DNS name).
+
+The `Zone:` field at the top represents the zone you're game is running in. When you save your score, it will automatically save the
+zone you were playing in and display it in the High Score list.
+
+See who can get the highest score!
+
+## Migrate Pac-Man Application to Cluster B (Manually)
+
+### Save cluster resources in pacman namespace
+
+Create dump directory:
+
+```bash
+mkdir ./pacman-ns-dump
+```
+
+Export desired namespace:
+
+```bash
+kubectl get --export -o=json ns | jq '.items[] |
+select(.metadata.name=="pacman") |
+del(.status,
+        .metadata.uid,
+        .metadata.selfLink,
+        .metadata.resourceVersion,
+        .metadata.creationTimestamp,
+        .metadata.generation
+    )' > ./pacman-ns-dump/ns.json
+
+```
+
+Export desired resources from namespace:
+
+```bash
+for ns in $(jq -r '.metadata.name' < ./pacman-ns-dump/ns.json);do
+    echo "Namespace: $ns"
+    kubectl --namespace="${ns}" get --export -o=json pvc,secrets,svc,deploy,rc,ds | \
+    jq '.items[] |
+        select(.type!="kubernetes.io/service-account-token") |
+        del(
+            .spec.clusterIP,
+            .metadata.uid,
+            .metadata.selfLink,
+            .metadata.resourceVersion,
+            .metadata.creationTimestamp,
+            .metadata.generation,
+            .metadata.annotations,
+            .status,
+            .spec.template.spec.securityContext,
+            .spec.template.spec.dnsPolicy,
+            .spec.template.spec.terminationGracePeriodSeconds,
+            .spec.template.spec.restartPolicy,
+            .spec.storageClassName,
+            .spec.volumeName
+        )' > "./pacman-ns-dump/pacman-ns-dump.json"
+done
+```
+
+### Switch Contexts to Cluster B (Destination)
```

Comment:
These instructions below are still referencing the GKE us-west and GKE us-central clusters.